package/kernel/mac80211/ath.mk

```diff
@@ -225,7 +225,7 @@ define KernelPackage/ath9k/config
 		bool "Add wireless noise as source of randomness to kernel entropy pool"
 		depends on PACKAGE_kmod-ath9k
 		select PACKAGE_kmod-random-core
-		default n
+		default y
```

Comment:
I think OpenWrt core members would be interested in how much does it make a difference in storage capacity.